integration/experiment/monitor/run_monitor_hpl_cpu.py

```diff
@@ -49,10 +49,12 @@
     parser.add_argument('--perc-dram', dest='perc_dram_per_node',
                     action='store', type=float, default=0.9,
                     help='Ratio of the total node DRAM that should be used for the HPL matrix (assuming DP). Default is 0.9.')
+    parser.add_argument('--codebase', action='store', type=str, default="netlib", choices=["mkl", "netlib"],
+                    help='Choose mkl or netlib based on the version of HPL to run. Default is netlib.')
```

Comment:
indentation here and above

Reply:
> My initial reaction is that is we have options for both the build and run, perhaps these two versions should be separate AppConf classes (with some shared code). Rather than have an option to the run script, there would be separate run scripts.

The MKL and and netlib versions run with the same exact set of parameters but their executable path are different and there are slight differences in setup. One could argue that we could use OOP to extend the base class, it's a choice. Is it problematic to keep it this way?

Reply:
Also netlib was the starting version however I can see that it performs very poorly compared to MKL so I made MKL default.